app/controllers/schools/courses_controller.rb

```diff
@@ -223,6 +223,20 @@ def evaluation_criteria
         authorize(scope.find(params[:id]), policy_class: Schools::CoursePolicy)
     end
 
+    # GET /courses/:id/clone
+    def clone
+      course = authorize(
+        scope
+          .where(id: params[:id])
+          .includes(:evaluation_criteria, :levels, :target_groups, :targets)
+          .first,
+        policy_class: Schools::CoursePolicy
+      )
+      new_name = [course.name, "copy"].join(" - ")
+      ::Courses::CloneService.new(course).clone(new_name, course.school)
+      render 'index'
```

Comment:
Right now, after clicking the clone button, the editor for the course _that was cloned_ opens up, which is confusing. When deferred, this should probably be a message telling the admin that the operation will be completed in a short while. Flash messages work in school admin routes:

```ruby
flash[:notice] = I18n.t('courses.clone.queued')
redirect_to school_courses_path
```